express-delete/index.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const app = express();
+
+// console.log(grades)
+
+app.delete('/api/grades/:id', (req, res) => {
+
+  delete grades['12'];
+
+  // console.log(grades)
+  const gradeArray = [];
+  for (const id in grades) {
+    gradeArray.push(grades[id]);
+  }
+
+  // res.sendStatus(204)
+  res.json(gradeArray);
+  res.send(res.params);
```

Comment:
You can't send multiple things back from an endpoint, you can only send one thing back. And for delete you should just be sending back the status code `204`. To send back just a status code use `res.sendStatus(204)`